### PR TITLE
Modernize embulk-formatter-csv with v0.11-ready API/SPI

### PR DESCRIPTION
**Pull request by dmikurube**

### CODE REVIEW

File: embulk-formatter-csv/build.gradle [hunk 1]
```diff
@@ -62,6 +68,84 @@ javadocJar {
...
+
+                developers {
+                    developer {
+                        name = "Sadayuki Furuhashi"
+                        email = "frsyuki@gmail.com"
+                    }
+                    developer {
+                        name = "Hiroyuki Sato"
+                        email = "hiroysato@gmail.com"
+                    }
```
Comment (dmikurube) [inline embulk-formatter-csv/build.gradle:108]: @hiroyuki-sato We're going to separate `embulk-standards` plugins as individual plugins, then to release them as independent pure-Java Embulk plugin JARs into Maven Central. In this `embulk-formatter-csv`, we wanted to include you as a developer from the pull request #128. What do you think?

Comment (dmikurube) [reply embulk-formatter-csv/build.gradle:108]: Assuming that @hiroyuki-sato is okay with it as he's okay with https://github.com/embulk/embulk/pull/1391#discussion_r586286675

Comment (hiroyuki-sato) [reply embulk-formatter-csv/build.gradle:108]: Thank you! No problem. Sorry, I've missed this mention.

**Comment by dmikurube** [comment]
`embulk-util-file:0.1.3` would work. Updated.

**Review by huylenq — APPROVED** [review]
LGTM

**Comment by dmikurube** [comment]
Thanks!

**Pull request merged**